pkg/bgs

```diff
@@ -0,0 +1,21 @@
+[mirrors]
+https://github.com/Gottox/bgs/archive/master.tar.gz
+
+[vars]
+filesize=4991
+sha512=30c806c097a6fdf2afd9a17a2cc1c2a27cce65df17a33921952ac68946b2a0efbcabcff00fab67013989478844e00c17899b4036bbd69dd7a110789ee4b2622c
+pkgver=1
+tardir="bgs-master"
```

Comment:
after fixing the master download link, please also use the tarball directive to save it under a proper name